lib/screens/login/login_repository.dart

```diff
@@ -7,7 +7,7 @@ import 'package:ecellapp/core/utils/logger.dart';
 import 'package:http/http.dart' as http;
 
 abstract class LoginRepository {
-  //Takes in login credentials (here id and password)
+  /// Takes in login credentials (here id and password)
   //Use by implementing this class
```

Comment:
ig it was obvious, but the `///` applies to this too. Read the documentation I linked in the previous review.